runner.py

```diff
@@ -93,15 +93,12 @@ def parse_args():
     if not args.test:
         args.test = [config.PHRTOS_TEST_DIR]
 
-    if not args.target:
-        # Run on all available targets
-        args.target = config.ALL_TARGETS
-
-    if not args.serial:
-        args.serial = config.DEVICE_SERIAL_PORT
+    # the default one is DEVICE_SERIAL_PORT_NXP
+    if 'armv7a9-zynq7000-zedboard' == args.target and args.serial != config.DEVICE_SERIAL_PORT_XYLINX:
```

Comment:
Is weird, we allow to choose serial port but if we choose other that runner want to then we silently change it?